tox.ini

```diff
@@ -7,7 +7,7 @@ deps =
         pytest
 commands =
         python setup.py check -m -s
-        py.test {posargs}
+        pytest {posargs}
```

Comment:
pay attention also on tox.ini, `py.test` is deprecated since 3.0
https://docs.pytest.org/en/latest/changelog.html
> Introduce pytest command as recommended entry point. Note that py.test still works and is not scheduled for removal.

But I didn't find out why is stop working for me anymore..

Reply:
thanks for the catch! this is what I get for copying a tox.ini from somewhere 😬